recipes/klayout/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+"${SRC_DIR}/build.sh"
+echo "bin-release Contents"
+echo "--------------------"
+ls bin-release
```

Comment:
find? (if you want to make sure to list everything recursively?)

Reply:
I'll update that if necessary for debug.